packages/office-ui-fabric-react/src/components/Button/Button.Props.ts

```diff
@@ -293,7 +293,43 @@ export interface IButtonStyles {
   descriptionChecked?: IStyle;
 
   /**
-   * Style for the screen reader text.
+   * Style override for the screen reader text.
    */
   screenReaderText?: IStyle;
+
+  /**
+   * Style override for the container div around a SplitButton element
+   */
+  splitButtonContainer?: IStyle;
```

Comment:
Can we instead create a ISplitButtonStyles interface that encapsulates all split button styles?